.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
+max-line-length = 120
+max-complexity = 18
```

Comment:
The max-line-length with `black` should be 88. (see [documentation](https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#flake8))